src/cli/commands/force/lightning/local/setup.ts

```diff
@@ -22,18 +28,24 @@ export class Setup extends BaseCommand {
     // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
     public static readonly flags = {
         ...CommandLineUtils.createFlag(FlagsConfigType.JsonFlag, false),
+        ...CommandLineUtils.createFlag(FlagsConfigType.OutputFormatFlag, false),
         ...CommandLineUtils.createFlag(FlagsConfigType.LogLevelFlag, false),
         ...CommandLineUtils.createFlag(FlagsConfigType.ApiLevelFlag, false),
         ...CommandLineUtils.createFlag(FlagsConfigType.PlatformFlag, true)
     };
 
     protected _commandName = 'force:lightning:local:setup';
 
-    public async run(): Promise<void> {
+    public async run(): Promise<RequirementCheckResultType | void> {
         // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
         this.logger.info(`Setup command called for ${this.flagValues.platform}`);
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-argument
-        return RequirementProcessor.execute(this.commandRequirements, this.flagValues.json);
+        return RequirementProcessor.execute(this.commandRequirements, this.jsonEnabled());
+    }
+
+    // eslint-disable-next-line class-methods-use-this
+    protected override getOutputSchema<TSchema extends z.ZodTypeAny>(): TSchema | undefined {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-return
+        return RequirementCheckResultSchema as any;
     }
```

Comment:
I think it is better to move this to `BaseCommand` class instead of here. All of our commands extend from `BaseCommand` so if you put this there then all commands will inherit it automatically. And if any command needs to customize the behavior then they can just override it.